### ISSUE TITLE
Support NLP workloads

### ISSUE THREAD

**Issue by GuillaumeLeclerc** [libffcv/ffcv#83]
This includes:

- [ ] Show an example now how to pass text and padding (it should work with the code base as it is now)
- [ ] Integrate slow pytorch based HuggingFace Tokenizers
- [ ] Integrate fast Rust based HuggingFace Tokenizers

I definitely don't have the bandwidth to do that now but I'm keeping this issue if someone wants to work on it.

**Comment by andrewilyas** [comment]
@GuillaumeLeclerc were you thinking something like this for the first task? https://github.com/libffcv/ffcv/issues/57#issuecomment-1016529098 If so I can clean it up and put it in a more prominent location (could maybe make a page about it in the docs?).

**Comment by GuillaumeLeclerc** [comment]
I don't think this example shows how to get the actual length and pass it to the model right ?
